.sandbox/provision/lifecycle-asg/common.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+function redirect_output() {
+    local dest="$1"
+    exec 3>&1 4>&2
+    exec &> $dest
+    set -ex
+}
+
+function restore_output() {
+    set +x
```

Comment:
Usually, we put `set -ex` at the beginning of the script. It's ok to have `x` on as it's printed on stderr.